golang/Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY bashrc /root/.bashrc
 
 RUN go install github.com/codegangsta/gin@latest && \
     go install github.com/go-delve/delve/cmd/dlv@latest && \
-    go install golang.org/x/tools/gopls@latest && \
-    curl -sSfL https://raw.githubusercontent.com/cosmtrek/air/master/install.sh | sh -s -- -b /usr/bin -d v1.42.0
+    go install golang.org/x/tools/gopls@latest
+RUN curl -sSfL https://raw.githubusercontent.com/air-verse/air/master/install.sh | sh -s -- -b /usr/bin
```

Comment:
Update installation instructions for air.
https://github.com/air-verse/air?tab=readme-ov-file#via-installsh